code/modules/uplink/uplink_items.dm

```diff
@@ -220,6 +220,15 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	surplus = 0
 	include_modes = list(/datum/game_mode/nuclear)
 
+/datum/uplink_item/dangerous/rawketlawnchair
+	name = "84mm Rocket Propelled Grenade Launcher"
+	desc = "A reusable rocket propelled grenade launcher preloaded with a low-yield 84mm SRM-8 HE round. \
```

Comment:
srm-8 stands for short range missle x 8
its not the actual name for a projectile, its a name for the weapon

SRM-20 would be a huge rack with 20 short range missles

https://mwo.gamepedia.com/Short_Range_Missile_6

Reply:
https://github.com/tgstation/tgstation/blob/1b50671ca072dd646e98bee971a6d20d5c06a2de/code/modules/projectiles/projectile/special/rocket.dm#L32-L33

?

Reply:
its not like you ever see projectiles anyway, it was probably named like that to avoid confusion

Reply:
renaming things to just SRM would be correct with the refrence of the items

http://www.sarna.net/wiki/Short_Range_Missile

Reply:
the projectile is referred to as the SRM-8 rocket because it gets fired from a SRM-8 and nothing else